core/src/main/scala/sttp/tapir/Mapping.scala

```diff
@@ -65,12 +65,26 @@ object Mapping {
     * otherwise an error is reported.
     */
   def stringPrefixCaseInsensitive(prefix: String): Mapping[String, String] = {
+    def removePrefix(v: String): DecodeResult[String] = cropPrefix(v, prefix)
+
+    Mapping.fromDecode(removePrefix)(v => s"$prefix$v")
```

Comment:
I think you should be able to simply write `Mapping.fromDecode(cropPrefix(_, prefix)(...)`?

Reply:
unless that's intentional for readability :)